### PR TITLE
Added links to multiple Socketcluster Clients

### PR DESCRIPTION
**Pull request by sacOO7**
- I have added links to Ruby, C, C++ and **Unity client** (Should be ready within 2 weeks)
- Out of which, we will release stable version of Ruby and C in 2-3 weeks.
- Unity client will be same as DotNet client, only underlying Json encoding library will change from NewtonSoft to LitJson.
- The UI might seem odd, since not whole two rows are filled with SDK's. But, it should work. We can discuss if it is looking too odd.

### CODE REVIEW

File: public/app/views/index.html [hunk 1]
```diff
@@ -163,6 +163,43 @@ <h5 style="margin-top: 20px; margin-bottom: 40px"><strong>GO client</strong></h5
...
                 <!--<p class="text-muted">Each Bootstrap theme includes all the source and compiled files you'll need, including npm and Gulp based build tools.</p>-->
             </div>
+
+            <div class="col-sm-2">
+                <a href="https://github.com/OpenSocket/socketcluster-client-ruby">
+                    <img class="homepage-grid-icon" src="https://image.flaticon.com/icons/svg/919/919842.svg"
+                         style="width: 25%; height: 25%">
+                </a>
+                <h5 style="margin-top: 20px; margin-bottom: 40px"><strong>Ruby client (Beta) </strong></h5>
+                <!--<p class="text-muted">Each Bootstrap theme includes all the source and compiled files you'll need, including npm and Gulp based build tools.</p>-->
```
Comment (MegaGM) [inline public/app/views/index.html]: Why did you include all those HTML comments? :slightly_smiling_face:

Comment (sacOO7) [reply public/app/views/index.html]: Ok I will remove it....

**Comment by sacOO7** [comment]
Hi @MegaGM @jondubois any updates on this?

**Comment by sacOO7** [comment]
Guys, any updates on this???????

**Pull request merged**